documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -45,8 +45,8 @@ For example, when you change service account labels or annotations using the `te
 The `UseStrimziPodSets` feature gate has moved to GA, which means it is now permanently enabled and cannot be disabled.
 Support for `StatefulSets` has been removed and Strimzi is now always using `StrimziPodSets` to manage Kafka and ZooKeeper pods.
 
-IMPORTANT: Downgrade to Strimzi 0.27 and earlier versions is not possible when using `StrimziPodSets`.
-If you need to downgrade to Strimzi 0.27 or earlier, you have to first downgrade to Strimzi 0.28 - 0.34, disable the `USeStrimziPodSets` feature gate and only then downgrade to Strimzi 0.27 or earlier.
+IMPORTANT: With the `UseStrimziPodSets` feature gate permanently enabled, it is no longer possible to downgrade directly between Strimzi 0.27 or earlier and Strimzi 0.35 and newer.
```

Comment:
It seems a bit confusing with the older version first when you talk about downgrade. As that starts with the new version. Mybe this would be better?

```suggestion
IMPORTANT: With the `UseStrimziPodSets` feature gate permanently enabled, it is no longer possible to downgrade directly from Strimzi 0.35 and newer to Strimzi 0.27 or earlier.
```